app/schema.py

```diff
@@ -131,3 +131,32 @@ class Feedback(BaseModel):
 class Resource(BaseModel):
     name: constr(max_length=255)
     item_id: constr(max_length=255)
+
+
+class Comments(BaseModel):
+    comment_id: constr(max_length=255)
+    comment_text: Optional[constr(max_length=2000)]
+    created_at: datetime
+    note_id: constr(max_length=255)
+    updated_at: datetime
+
+
+class Notes(BaseModel):
+    note_id: constr(max_length=255)
+    created_by: constr(max_length=255)
+    status: Literal['in progress', 'resolved', 'no action needed', 'escalated']
+    content_type: constr(max_length=255)
+    content: constr(max_length=255)
+    level: constr(max_length=255)
+    visible_to_admin: Optional[bool]
+    visible_to_mentor: Optional[bool]
+    visible_to_mentee: Optional[bool]
+    created_at: datetime
+    updated_at: datetime
+    mentor_id: constr(max_length=255)
+    mentee_id: constr(max_length=255)
+
+
+class MenteeProgression(BaseModel):
```

Comment:
I believe we plan to scrap this collection entirely in the next step. Perhaps make a note in the docstring that we won't be keeping this.